### ISSUE TITLE
Cannot create new chats

### ISSUE THREAD

**Issue by probablytom** [darrenburns/TeamProject3#11]
There's a bug in creating new chats. Error reads:

    OperationalError at /projects/1/newchat/
    table chat_ticket has no column named due_date

Looks like it was introduced during the metadata work, maybe?

**Comment by probablytom** [comment]
Related: Given the number of problems we've been having with migrations lately, introducing South to the project might be a wise move. Reading online, almost all of the solutions to this issue are "Use South"...

**Comment by probablytom** [comment]
OKAY. After a little poking around: 

When Django makes its migrations, it doesn't (doesn't always?) make changes to table structures, it only adds tables as required. I'm not convinced this is true, it's what I've gleamed from a cursory googling, but the point is that migrations aren't picking up changes made in the metadata work done earlier. 

When we try to make new conversations and add them to a previously existing database that's had migrations applied, Django gets confused because the migrations haven't picked up some of the changes made in the models. 

There are three ways I can see to fix this: burn all old databases, write our own additional migrations for the changes we've made so far, or adopt South. Not sure South would help at this juncture, now that we've already made the changes and messed with the migrations - no idea how it works -- but it'd prevent this from happening in the future. 

I'll write the migration in the labs tomorrow to fix this temporarily.
